com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -186,9 +186,14 @@ public bool TiledDeferredShading
         public int GBufferSpecularMetallicIndex { get { return 1; } }
         public int GBufferNormalSmoothnessIndex { get { return 2; } }
         public int GBufferLightingIndex { get { return 3; } }
-        //public int GbufferDepthIndex { get { return useRenderPass ? 4 : -1; } }
-        //public int GBufferSliceCount { get { return useRenderPass ? 5 : 4; } }
+        public int GBufferDepthIndex { get { return 4; }}
+        public int GBufferAdditionalDepthIndex { get { return 5; }}
```

Comment:
I think there is a confusion here. GBufferAdditionalDepthIndex == GBufferDepthIndex. Before renderPass, GBuffer does not need to output depth a color.

Reply:
removed the GBufferDepthIndex and created an independent depth buffer attachment descriptor instead